### PR TITLE
Removed unnecessary documentation from README.

### PR DESCRIPTION
**Pull request by bkuhlmann**
- Removed unused badges.
- Removed secure install notes from README.
- Removed versioning info from README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,7 @@
 # Tester
 
-[![Gem Version](https://badge.fury.io/rb/tester.svg)](http://badge.fury.io/rb/tester)
 [![Code Climate GPA](https://codeclimate.com/github/bkuhlmann/tester.svg)](https://codeclimate.com/github/bkuhlmann/tester)
 [![Code Climate Coverage](https://codeclimate.com/github/bkuhlmann/tester/coverage.svg)](https://codeclimate.com/github/bkuhlmann/tester)
-[![Gemnasium Status](https://gemnasium.com/bkuhlmann/tester.svg)](https://gemnasium.com/bkuhlmann/tester)
-[![Travis CI Status](https://secure.travis-ci.org/bkuhlmann/tester.svg)](https://travis-ci.org/bkuhlmann/tester)
-[![Patreon](https://img.shields.io/badge/patreon-donate-brightgreen.svg)](https://www.patreon.com/bkuhlmann)
```
Comment (erikmingo) [inline README.md]: asdfasdfasdfasdfasdf

File: README.md [hunk 2]
```diff
@@ -33,15 +29,7 @@
...
 # Setup
 
-For a secure install, type the following (recommended):
-
-    gem cert --add <(curl -Ls https://www.my-website.com/gem-public.pem)
-    gem install tester --trust-policy MediumSecurity
-
-NOTE: A HighSecurity trust policy would be best but MediumSecurity enables signed gem verification while
-allowing the installation of unsigned dependencies since they are beyond the scope of this gem.
-
```
Comment (erikmingo) [inline README.md]: huh

File: README.md [hunk 3]
```diff
@@ -57,14 +45,6 @@ To test, run:
...
 
-# Versioning
-
-Read [Semantic Versioning](http://semver.org) for details. Briefly, it means:
-
-- Patch (x.y.Z) - Incremented for small, backwards compatible bug fixes.
-- Minor (x.Y.z) - Incremented for new, backwards compatible public API enhancements and/or bug fixes.
-- Major (X.y.z) - Incremented for any backwards incompatible public API changes.
-
 # Code of Conduct
```
Comment (erikmingo) [inline README.md]: adf

File: README.md (hunk 1 above)
Comment (bkuhlmann) [inline README.md]: Thanks, [Fixed](https://github.com/kapost/tester/commit/dc1f9f0465d1ab6f2b53258156114d27dc031fb2).

**Comment by derekclee** [comment]
:+1:

**Pull request merged**